.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+on: pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm i
+        shell: bash
+      - name: Check TypeScript
+        run: npx tsc --noEmit
+        shell: bash
+      - name: Run ESLint
+        run: npm run lint
```

Comment:
Does this treat warnings as errors?

Reply:
No

Reply:
We can define which rules will cause errors in the eslintrc file